dpctl/__init__.py

```diff
@@ -1,27 +1,27 @@
-##===----------------- __init__.py - dpctl module -------*- Cython -*------===##
-##
-##                      Data Parallel Control (dpCtl)
-##
-## Copyright 2020 Intel Corporation
-##
-## Licensed under the Apache License, Version 2.0 (the "License");
-## you may not use this file except in compliance with the License.
-## You may obtain a copy of the License at
-##
-##    http://www.apache.org/licenses/LICENSE-2.0
-##
-## Unless required by applicable law or agreed to in writing, software
-## distributed under the License is distributed on an "AS IS" BASIS,
-## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-## See the License for the specific language governing permissions and
-## limitations under the License.
-##
-##===----------------------------------------------------------------------===##
-##
-## \file
-## This top-level dpctl module.
-##
-##===----------------------------------------------------------------------===##
+#===----------------- __init__.py - dpctl module -------*- Cython -*--------===#
```

Comment:
I do not like this line. It duplicates file name and also it lies about Cython. Conclusion: unnecessary support costs.
Lets keep only License at the header of the file and docstring with description of the content.

Reply:
Cython was an obvious mistake, should be Python. I was following the convention LLVM uses, but we can make the change.